src/modules/UI/scenes/Settings/CurrencySettings.ui.js

```diff
@@ -9,10 +9,13 @@ import styles from './style'
 import {border as b} from '../../../utils'
 import Row from './components/Row.ui.js'
 import RadioRows from './components/RadioRows.ui.js'
+import RowRoute from './components/RowRoute.ui'
 
 import type {GuiDenomination} from '../../../../types'
 
 const SETTINGS_DENOMINATION_TEXT = s.strings.settings_denominations_title
+const SETTINGS_OPTIONS_TEXT      = s.strings.settings_options_title
```

Comment:
These additional levels of indirection for strings is unnecessary. Please remove.

Reply:
I dont know what that means